rust/ql/test/query-tests/diagnostics/ExtractionErrors.expected

```diff
@@ -0,0 +1,6 @@
+| does_not_compile.rs:2:6:2:5 | expected SEMICOLON | Extraction failed in does_not_compile.rs with error expected SEMICOLON | 2 |
+| does_not_compile.rs:2:9:2:8 | expected SEMICOLON | Extraction failed in does_not_compile.rs with error expected SEMICOLON | 2 |
+| does_not_compile.rs:2:13:2:12 | expected SEMICOLON | Extraction failed in does_not_compile.rs with error expected SEMICOLON | 2 |
+| does_not_compile.rs:2:21:2:20 | expected SEMICOLON | Extraction failed in does_not_compile.rs with error expected SEMICOLON | 2 |
+| does_not_compile.rs:2:26:2:25 | expected SEMICOLON | Extraction failed in does_not_compile.rs with error expected SEMICOLON | 2 |
+| does_not_compile.rs:2:32:2:31 | expected field name or number | Extraction failed in does_not_compile.rs with error expected field name or number | 2 |
```

Comment:
I'm surprised we don't get any results from `error.rs` (which contains `compile_error!("An error!");`).

Reply:
The line in error.rs isn't a syntax error.

Reply:
I suppose.  I feel like it should show up somewhere (and is a cleaner test than `does_not_compile.rs`).

Reply:
The extractor currently only parses the code, and `error.rs` does not contain a syntax error. 
I would expect an error to be reported once we implement macro expansion.

Reply:
Sounds good to me. :+1: